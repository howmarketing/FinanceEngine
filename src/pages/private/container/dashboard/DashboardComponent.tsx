import React, { useEffect, useState } from "react";
import { useForm } from "react-hook-form";
import ButtonComponent from "shared/components/button/ButtonComponent";
import CircleChartComponent from "../../components/CircleChartComponent";
// Styles
import "./DashboardComponent.scss";

const DashboardComponent: React.FC<any> = () => {
  return (
    <>
      <div className="header-container">
        <h1 className="container-title">Tela Inicial</h1>
        {/* <ButtonComponent value="texto" action={() => 'text'} /> */}
        <CircleChartComponent />
      </div>
    </>
  );
};

export default DashboardComponent;
