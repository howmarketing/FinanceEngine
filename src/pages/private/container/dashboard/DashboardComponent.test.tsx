import React from 'react';
import { render } from '@testing-library/react';
import TestRenderer from 'react-test-renderer';
import DashboardComponent from './DashboardComponent';

describe('dashboard Component', () => {
	it('renders component', () => {
		render(<DashboardComponent />);
		expect(DashboardComponent).toBeTruthy();
	});
});

it('dashboard Component render snapshot', () => {
	const tree = TestRenderer.create(<DashboardComponent />).toJSON()
	expect(tree).toMatchSnapshot()
})
