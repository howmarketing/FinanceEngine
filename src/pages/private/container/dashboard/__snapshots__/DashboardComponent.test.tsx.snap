// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard component render 1`] = `
Array [
  <div
    className="header-container"
  >
    <h1
      className="container-title"
    >
      Dashboard
    </h1>
  </div>,
  " ",
]
`;

exports[`dashboard Component render snapshot 1`] = `
Array [
  <div
    className="header-container"
  >
    <h1
      className="container-title"
    >
      Dashboard
    </h1>
  </div>,
  " ",
]
`;
